pkg/config/storage.go

```diff
@@ -0,0 +1,74 @@
+package config
+
+// StorageConfig provides configs for various storage backends
+type StorageConfig struct {
+	CDN   *CDNConfig   `validate:""`
```

Comment:
do we need to put `validate:""`?? can we not just omit it? same as below.

Reply:
It ends up skipping properties that don't have that tag, and since we want to run the validation for CDNConfig, MongoConfig, etc, we need it here!